dev/docker/docker-compose.yml

```diff
@@ -32,7 +32,7 @@ x-chrome: &chrome
 
 
 x-php: &php
-  image: moderntribe/squareone-php:74-2.1.1
+  image: moderntribe/squareone-php:74-2.1
```

Comment:
Is this the right order? It looks like we're going backwards.

Reply:
Yeah, it will always grab `74-2.1.x`: https://hub.docker.com/layers/moderntribe/squareone-php/74-2.1/images/sha256-10762c1dd18858d8a7f59c3bf1efa2871b922c3154822b2b14e3d92e0b65634b?context=explore